creator-node/scripts/start.sh

```diff
@@ -64,9 +67,18 @@ if [ -z "$dbUrl" ]; then
 fi
 
 if [[ "$devMode" == "true" ]]; then
-    ./node_modules/.bin/nodemon --inspect=0.0.0.0:${debuggerPort} src/index.js | tee >(logger) | ./node_modules/.bin/bunyan
+    if [ "$link_libs" = true ]
```

Comment:
```suggestion
    if [ "$link_libs" = true ]; then
```

Reply:
when is `link_libs` true? it seems set to `false` at start of file

Reply:
in very specific dev scenarios we set link_libs to true, usually if we're testing changes that involve both cnode + libs

Reply:
gotcha! thank you for this clarification!